opentelemetry/chart/values.yaml

```diff
@@ -1,17 +1,33 @@
 # SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
 # SPDX-License-Identifier: Apache-2.0
 
+owner-info:
+  service: otel
+  support-group: observability
+  maintainers:
+  - name: timojohlo
+  - name: kuckkuck
+  - name: viennaa
+  helm-chart-url: https://github.com/sapcc/helm-charts/tree/master/system/opentelemetry-operator
+
 opentelemetry-operator:
   crds:
-    create: true
+    create: false
   admissionWebhooks:
     failurePolicy: 'Ignore'
+    create: true
+    certManager:
+      enabled: false
+    autoGenerateCert:
+      enabled: true
+      recreate: false
   manager:
     collectorImage:
-      repository: ghcr.io/cloudoperators/opentelemetry-collector-contrib
+      repository: keppel.eu-de-1.cloud.sap/ccloud/opentelemetry/collector-image
       #repository: ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-contrib
-      tag: main
       #tag: 0.104.0
+      #tag: main
+      tag: latest
```

Comment:
can we go with versions from Chart.yaml here? same applies to opentelemetry-operator